src/components/common/PostalCode/postal-code.scss

```diff
@@ -0,0 +1,50 @@
+@import "src/styles/scaffold";
+
+.postal-code-input {
+  // Lighthouse
+  margin-top: 1rem;
+
+  [data-store-label] {
+    font-family: Arial; /* stylelint-disable-line */ // Prevent layout shift
+  }
+
+  [data-store-input] {
+    border: var(--fs-border-width-default) solid var(--fs-border-color-default);
+    border-radius: var(--fs-border-radius-default);
+
+    // Lighthouse
+    height: var(--fs-control-tap-size); /* stylelint-disable-line */
+    margin-left: 1rem;
+    font-family: Arial; /* stylelint-disable-line */ // prevent layout shift
+  }
+}
```

Comment:
I know this is the contents from `src/components/common/PostalCode/postal-code-input.scss` so it doesn't bother me now. It should be updated when we style in input inside the regionalization modal.